build/dependencies.props

```diff
@@ -332,56 +331,56 @@
-    <ExternalDependency Include="System.Security.Principal.Windows" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.ServiceProcess.ServiceController" Version="4.5.0-preview1-25914-04" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.Text.Encoding.CodePages" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.Text.Encodings.Web" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.Threading.AccessControl" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.Threading.Tasks.Dataflow" Version="4.8.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.Threading.Tasks.Extensions" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
-    <ExternalDependency Include="System.ValueTuple" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
+    <ExternalDependency Include="System.Security.AccessControl" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Security.Cryptography.Cng" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Security.Cryptography.Pkcs" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Security.Cryptography.ProtectedData" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Security.Cryptography.Xml" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Security.Permissions" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Security.Principal.Windows" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.ServiceProcess.ServiceController" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Text.Encoding.CodePages" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Text.Encodings.Web" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Threading.AccessControl" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Threading.Tasks.Dataflow" Version="4.9.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Threading.Tasks.Extensions" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.ValueTuple" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
```

Comment:
Can reorder the lines to keep the grouping by package feed?

Reply:
This seems like a bunch of work for a file that's meant to be auto managed. Would we change it back once we ship 2.1.0?

Reply:
The file isn't meant to be automanaged, at least not in its current form.

Reply:
Are you manually updating versions of external dependencies? Again, just sounds like a bunch of unnecessary work.

Reply:
Yes. Hence #577

Reply:
Cool. More of a reason to stop manually curating the order.